SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1860,6 +1879,28 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			WellKnownEntitySystem->Advance();
 		}
 
+		if (SpatialDebugger != nullptr)
```

Comment:
Shouldn't the SpatialDebugger also use the actorsubview?
Otherwise the debugger could attempt to refer to actors that aren't present.

Reply:
It should. I didn't want to add that to this PR. It's fairly bloated as is- but I can do it if you feel it's necessary

Reply:
That's fine, but can we add a ticket please